### PR TITLE
Pandoc extracts images from files and uploads them to media

### PR DESCRIPTION
**Pull request by arunabh98**

### CODE REVIEW

File: oksp/settings/base_settings.py [hunk 1]
```diff
@@ -95,6 +96,8 @@
 
 USE_TZ = True
 
+MEDIA_URL = '/media/'
```
Comment (udiboy1209) [inline oksp/settings/base_settings.py]: Dont make a new variable, we have `MEDIA_ROOT` declared right below which is the same.

File: docupload/htmlify.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import os
 
 import pypandoc
+from django.conf import settings
```
Comment (udiboy1209) [inline docupload/htmlify.py]: unused import ?

**Comment by udiboy1209** [comment]
Fixes #6

**Comment by arunabh98** [comment]
Did the necessary fixes.

File: docupload/htmlify.py [hunk 3]
```diff
@@ -1,23 +1,41 @@
...
-    html = pypandoc.convert(tmp_loc, 'html5')
+    html = pypandoc.convert( 
+        tmp_loc,
+        'html5',
+        extra_args=['--extract-media=']
+        )
+    img_counter = 1
+    soup = BeautifulSoup(html, 'lxml')
+    for img in soup.findAll('img'):
+        img_name = '/media/' + str(doc_file) + '_' +str(img_counter) + '.jpg'
```
Comment (udiboy1209) [inline docupload/htmlify.py]: `str(doc_file)` makes the image name contain the doc file extension too for eg. I get the image name as `/media/test2.docx_1.jpg` which is not desirable. I guess you are too confused about what to do. I will modify it and merge today.

Comment (arunabh98) [inline docupload/htmlify.py]: Ok thanks
I could have made the image name just test2_1.jpg but then I thought it wouldn't make a difference to include the file extension also.
But whatever you feel would be the best you can modify it and merge

Comment (udiboy1209) [inline docupload/htmlify.py]: also, you assumed `jpg`, which is not the case everytime

Comment (arunabh98) [inline docupload/htmlify.py]: Yeah I know but even if the image was in PNG or gif it worked every time
But I will try changing that to include the desired extensions

Comment (arunabh98) [inline docupload/htmlify.py]: Fixed ext of images.

**Comment by udiboy1209** [comment]
Merged in [7a3fd4](https://github.com/stabiitb/oksp/commit/7a3fd4efa93eb835ce30ac12431a238f2aaf62d4)! Thanks for this contribution :smile:!

**Pull request closed without merging**